zerver/openapi/zulip.yaml

```diff
@@ -10479,6 +10479,8 @@ paths:
                     }
                 display_in_profile_summary:
                   description: |
+                    [Optional]: If not provided, defaults to `false`.
```

Comment:
I'm pretty sure there's an `required` OpenAPI tag that we across this file and should be used instead.